scripts/ci/check_patch.sh

```diff
@@ -3,10 +3,13 @@
 set -x
 set -e
 
-pip install Pygments collective.checkdocs pre-commit 
+pip install Pygments collective.checkdocs pre-commit
 
 # stop the build if there are any readme formatting errors
 python setup.py checkdocs
 
 # stop the build if there are any formatting errors
-pre-commit run --all-files
+err=0
+pre-commit run --all-files black || err=1
+pre-commit run --all-files flake8 || err=1
```

Comment:
I don't really like this as it's not future-proof (if more pre-commit hooks are added in `.pre-commit-config.yaml` in future, they'll have to be manually added here too. Ideally we need something like `pre-commit run --all-files --exclude-hooks shfmt`

Reply:
outdated